.github/test_fork12_pessimistic_e2e_args.json

```
@@ -0,0 +1,18 @@
+{
+  "deployment_stages": {
+    "deploy_aggkit_node": true,
+    "deploy_cdk_bridge_infra": false
+  },
+  "args": {
+    "zkevm_contracts_image": "jhkimqd/zkevm-contracts:v10.0.0-rc.6-fork.12",
```

There is even newer image:

```suggestion
    "zkevm_contracts_image": "jhkimqd/zkevm-contracts:v10.1.0-rc.5-fork.12",
```